source/tyr/tests/integration/v1_test.py

```diff
@@ -102,7 +115,9 @@ def check_v1_response(endpoint, request=None):
     resp_v1 = api_get('/v1/{}'.format(request))
     assert type(resp_v1) == dict
     assert endpoint in resp_v1
-    assert sorted(resp_v1[endpoint]) == sorted(resp_v0)
+    if isinstance(resp_v0, list) and isinstance(resp_v1[endpoint], list):
+        assert compare_responses(resp_v1[endpoint], resp_v0)
+    assert resp_v1[endpoint] == resp_v0
```

Comment:
maybe an else here ? Or do you want to perform both the check `compare_response(resp_v1[endpoint], resp_v0)` and `resp_v1[endpoint] == resp_v0` ?

Reply:
No, you're right, there should be an `else` 
It currently works because in py2, there's no need to sort the lists so both checks are ok.